graphql/schema.graphql

```diff
@@ -580,6 +580,7 @@ type TaskTestResult {
 
 type TestResult {
   id: String!
+  groupID: String
```

Comment:
@julianedwards do we ever expect groupID to be nil or an empty string?

Reply:
Yes in cases where a test is not grouped.

Reply:
I left it nil since groupid wont be populated if we end up fetching normal test results.